tests/vmi_networking_test.go

```diff
@@ -190,9 +188,25 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			)
 			log.Log.Infof("%v", output)
 			Expect(err).ToNot(HaveOccurred())
-			// the following substring is part of 'ip address show' output
-			expectedMtuString := fmt.Sprintf("mtu %d", expectedMtu)
-			Expect(strings.Contains(output, expectedMtuString)).To(BeTrue())
+
+			// The MTU of the Pod network varies, depending on the environment in use. We want to
+			// verify that the actual MTU is from the possible range, which is {1500, 1450}, minus
+			//  50 bytes for vxlan overhead.
+			possibleMtus := []int{1400, 1450}
+			mtuMatch := false
+			expectedMtuString := ""
+
+			for _, expectedMtu := range possibleMtus {
```

Comment:
I guess we could find that out once and don't do it every time, but the change is great.